src/lib/ui-switch/ui-switch.component.html

```diff
@@ -2,6 +2,7 @@
   type="button"
   class="switch"
   role="switch"
+  [disabled]="disabled"
```

Comment:
```suggestion
  [attr.disabled]="disabled"
```

Reply:
I'll give this change a try, but I don't think it works with the disabled attribute. I've had issues in the past where you end up with `disabled="false"` in the DOM, and the element is actually disabled.